test/Microsoft.OpenApi.SmokeTests/ApiGurus.cs

```diff
@@ -0,0 +1,107 @@
+
+        private readonly ITestOutputHelper output;
+    
+        public ApisGuruTests(ITestOutputHelper output)
+        {
+           
+            this.output = output;
+        }
+
+        static ApisGuruTests()
+        {
+            _httpClient.DefaultRequestHeaders.AcceptEncoding.Add(new System.Net.Http.Headers.StringWithQualityHeaderValue("gzip"));
+            _httpClient.DefaultRequestHeaders.UserAgent.Add(new System.Net.Http.Headers.ProductInfoHeaderValue("OpenApi.Net.Tests", "1.0"));
+           
+        }
+
+        public static IEnumerable<object[]> GetSchemas()
+        {
+
+            var listJsonStr = _httpClient
+                    .GetStringAsync("https://api.apis.guru/v2/list.json")
+                    .GetAwaiter().GetResult();
+
+                var json = JObject.Parse(listJsonStr);
+                foreach (var item in json.Properties())
+                {
+                    var versions = GetProp(item.Value, "versions") as JObject;
+                    if (versions == null)
+                        continue;
+                    foreach (var prop in versions.Properties())
+                    {
+                        var urlToJson = GetProp(prop.Value, "swaggerUrl")?.ToObject<string>();
+                        if (urlToJson != null)
+                            yield return new object[] { urlToJson };
+
+                        var utlToYaml = GetProp(prop.Value, "swaggerYamlUrl")?.ToObject<string>();
+                        if (utlToYaml != null)
+                            yield return new object[] { utlToYaml };
+                    }
+            }
+
+            JToken GetProp(JToken obj, string prop)
+            {
+                if (!(obj is JObject jObj))
+                    return null;
+                if (!jObj.TryGetValue(prop, out var jToken))
+                    return null;
+                return jToken;
+            }
+        }
+
+
+        [Theory(DisplayName = "APIs.guru")]
+        [MemberData(nameof(GetSchemas))]
+        public async Task EnsureThatICouldParse(string url)
+        {
+            var stopwatch = new Stopwatch();
+            
+            var response = await _httpClient.GetAsync(url);
+            if (!response.IsSuccessStatusCode)
+            {
+                output.WriteLine($"Couldn't load {url}");
+                return;
+            }
+            await response.Content.LoadIntoBufferAsync();
+            var stream = await response.Content.ReadAsStreamAsync();
+
+            stopwatch.Start();
+
+            var reader = new OpenApiStreamReader(new OpenApiReaderSettings() {
+            });
+            var openApiDocument = reader.Read(stream, out var diagnostic);
+
+            if (diagnostic.Errors.Count > 0)
+            {
+                output.WriteLine($"Errors parsing {url}");
+                output.WriteLine(String.Join("\n", diagnostic.Errors));
+ //               Assert.True(false);  // Uncomment to identify descriptions with errors.
```

Comment:
remove

Reply:
I'd like to leave that.  Once we ensure that the smoketests are not run during the automatic build, I think we might re-enable this.